src/ImageOptimizerServiceProvider.php

```diff
@@ -25,5 +25,7 @@ public function boot()
     public function register()
     {
         $this->mergeConfigFrom(__DIR__.'/../config/image-optimizer.php', 'image-optimizer');
+
+        $this->app['router']->aliasMiddleware('optimizeImages', \Spatie\LaravelImageOptimizer\Middlewares\OptimizeImages::class);
```

Comment:
This is a kinda of breaking. Current users of this package which did opt to not use the middleware will suddenly have the middleware installed.

Could you remove this change and restore the readme to it's current state?